backend/main/shared/flow.go

```diff
@@ -264,18 +287,60 @@ func (fa *FlowAdapter) EnforceTokenThreshold(creatorAddr string, c *Contract) (b
 	return true, nil
 }
 
-// Fungible Token Address here is hardcoded to emulator address, this should
-// be set based on environment
-func (fa *FlowAdapter) ReplaceContractPlaceholders(code string, c *Contract) []byte {
-	fungibleTokenAddr := fa.Config.Contracts["FungibleToken"].Aliases[fa.Env]
+func (fa *FlowAdapter) GetNFTIds(voterAddr string, c *Contract) ([]interface{}, error) {
+	flowAddress := flow.HexToAddress(voterAddr)
+	cadenceAddress := cadence.NewAddress(flowAddress)
+
+	script, err := ioutil.ReadFile("./main/cadence/scripts/get_nfts_ids.cdc")
+	if err != nil {
+		log.Error().Err(err).Msgf("error reading cadence script file")
+		return nil, err
+	}
+
+	script = replaceContractPlaceholders(string(script[:]), c, false)
 
-	code = placeholderFungibleTokenAddr.ReplaceAllString(code, fungibleTokenAddr)
+	cadenceValue, err := fa.Client.ExecuteScriptAtLatestBlock(
+		fa.Context,
+		script,
+		[]cadence.Value{
+			cadenceAddress,
+		},
+	)
+	if err != nil {
+		log.Error().Err(err).Msg("error executing script")
+		return nil, err
+	}
+
+	value := CadenceValueToInterface(cadenceValue)
+
+	// we can cast cadence type [Uint64] as Go type []interface{}
+	// In the case where ids are of string type we need an if statement somewhere to handle
+	nftIds := value.([]interface{})
+
+	return nftIds, nil
+}
+
+// @TODO
+// Hard coded Addresses here are for emulator dev only this should
+// be set based on environment var
+func replaceContractPlaceholders(code string, c *Contract, isFungible bool) []byte {
```

Comment:
On the `main` branch this function looks like this:

```// Fungible Token Address here is hardcoded to emulator address, this should
// be set based on environment
func (fa *FlowAdapter) ReplaceContractPlaceholders(code string, c *Contract) []byte {
	fungibleTokenAddr := fa.Config.Contracts["FungibleToken"].Aliases[fa.Env]

	code = placeholderFungibleTokenAddr.ReplaceAllString(code, fungibleTokenAddr)
	code = placeholderTokenName.ReplaceAllString(code, *c.Name)
	code = placeholderTokenAddr.ReplaceAllString(code, *c.Addr)
	return []byte(code)
}```

Can you update this function to use the `flow.json` to resolve the address based on the environment?  will need to add the addresses for testnet & mainnet to `flow.json`. you should be able to find some of those in the `flow.json` in the VESSEL repo if necessary